src/ks-app/src/app/containers/popover/tooltips.component.html

```diff
@@ -0,0 +1,23 @@
+<h1>Tooltips</h1>
+<div class="row">
+    <div class="col-xs-12 col-lg-6">
+      <h4>Clarity UI Tooltip (Medium)</h4>
+      <p>
+        <a href="javascript://" role="tooltip" aria-haspopup="true" class="tooltip tooltip-md">
+          <clr-icon shape="bell" size="24"></clr-icon>
+          <span class="tooltip-content">Lorem ipsum sit</span>
+        </a>
+      </p>
+    </div>
+    <div class="col-xs-12 col-lg-6">
+      <h4>Clarity Angular Tooltip (Large)</h4>
+      <p>
+        <clr-tooltip>
+          <clr-icon clrTooltipTrigger shape="info-circle" size="24"></clr-icon>
```

Comment:
Should use actionable icons here (icons wrapped in a link)

Reply:
Will update.